### PR TITLE
Upgrade cluster-autoscaler to v1.12.0

### PR DESCRIPTION
**Pull request by mvisonneau**
This is a requirement for the newest version of the cluster autoscaler `v1.12.0`

### CODE REVIEW

File: stable/cluster-autoscaler/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 description: Scales worker nodes within autoscaling groups.
 icon: https://github.com/kubernetes/kubernetes/blob/master/logo/logo.png
 name: cluster-autoscaler
-version: 0.7.0
+version: 0.7.1
```
Comment (unguiculus) [inline stable/cluster-autoscaler/Chart.yaml]: Why don't you update the appVersion to 1.12.0?

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: stable/cluster-autoscaler/Chart.yaml (hunk 1 above)
Comment (mvisonneau) [reply stable/cluster-autoscaler/Chart.yaml]: I did not believe it was the purpose of this MR but I'm happy to add it if you want.

Comment (unguiculus) [reply stable/cluster-autoscaler/Chart.yaml]: Currently, the PR adds a requirement for `v1.12.0` but doesn't use `v1.12.0`.

Comment (mvisonneau) [reply stable/cluster-autoscaler/Chart.yaml]: sounds good, sorted 👍

**Review by unguiculus — APPROVED** [review]

**Pull request merged**